src/app/home/home.page.html

```diff
@@ -11,7 +11,9 @@
  
   <ion-button (click)="onGoToAttritubesBtnClick()" data-test="launchAttributesPageButton" fill="clear">Edit Your Attributes</ion-button>
   <ion-button (click)="onGoToNotificationBtnClick()" fill="clear">Notifications</ion-button>
-  <ion-button (click)="onGoToPermissionsBtnClick()" data-test="launchPermissionsPageButton" fill="clear">Edit Your Permissions</ion-button>
+  
+  <ion-button *ngIf="getUserName() === 'testuser' " (click)="onGoToPermissionsBtnClick()" fill="clear">Edit User Permissions</ion-button>
```

Comment:
All users with the ADMIN role, should be able to access the permissions page. You can either implement that now, or remove this *ngIf check (adding it back later, once you do implement that requirement).